train/train_process.py

```diff
@@ -115,6 +115,26 @@ def _train(self, model_path, training_games_dir):
 
         return model
 
+    def _compare_models(self, model1_path, model2_path):
+        compare_res_file = os.path.join(self.cfg["working_area"], "compare_result.json")
+
+        profile = "dev" if self.cfg["debug"] == "true" else "release"
+        subprocess.run([
+            "cargo", "run", "--profile", profile, "--bin",
+            self.cfg["training"]["compare"]["exec"], "--",
+            "--model1-path", model1_path,
+            "--model2-path", model2_path,
+            "--games-num", str(self.cfg["training"]["compare"]["games_num"]),
+            "--result-file", compare_res_file,
+            "--sim-count", str(self.cfg["mcts"]["sim_count"]),
+            "--explore-factor", str(self.cfg["mcts"]["explore_factor"]),
+            "--threads", str(self.cfg["training"]["compare"]["threads"])],
+            stderr=sys.stderr, stdout=sys.stdout, check=True)
+
+        with open(compare_res_file, "r") as res_file:
+            res = json.load(res_file)

```

Comment:
This file-based communication could be problematic if we have more than one train_process happening at once - but this shouldn't happen, right?

Reply:
for now, but we should use NamedTemporaryFile or something, i never manage to get it work easily